platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_bitbang_reset.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Script to unfreeze a stuck I2C controller, by bit-banging a STOP cycle on the bus
+
+bit_bang_recovery()
+{
+
+# Clear the ERRSTS
+pcisysfs.py --set --val 0xffffffff --offset 0x018 --res /sys/devices/pci0000\:00/0000\:00\:13.0/resource0
+
+#Enable I2C bit-banging
+pcisysfs.py --set --val 0x80000000 --offset 0x388 --res /sys/devices/pci0000\:00/0000\:00\:13.0/resource0
+
+while true;do
```

Comment:
Doesn't this mean it will retry until i2c bitbang succeed? Can we have some limitation of bitbang retry?

Reply:
Will change the retry condition.